docs/sources/next/using-k6-browser/recommended-practices/_index.md

```diff
@@ -11,3 +11,4 @@ This section presents some examples and recommended practices when working with
 - [Hybrid approach to performance](https://grafana.com/docs/k6/<K6_VERSION>/using-k6-browser/recommended-practices/hybrid-approach-to-performance)
 - [Page object model pattern](https://grafana.com/docs/k6/<K6_VERSION>/using-k6-browser/recommended-practices/page-object-model-pattern)
 - [Selecting elements](https://grafana.com/docs/k6/<K6_VERSION>/using-k6-browser/recommended-practices/selecting-elements)
+- [`sleep` and `wait*`](https://grafana.com/docs/k6/<K6_VERSION>/using-k6-browser/recommended-practices/sleep-and-wait)
```

Comment:
I'd suggest using human-readable text, also to be consistent with the other links in this list.

```suggestion
- [Simulating waits and avoiding flaky tests](https://grafana.com/docs/k6/<K6_VERSION>/using-k6-browser/recommended-practices/sleep-and-wait)
```

Reply:
Maybe even something like "Simulating user input delay"?

Reply:
I've updated it in 7dd3265c2211b6be2c4fab4c040a3ecef4f1d8c4. WDYT?